composer.json

```
@@ -45,14 +45,16 @@
     "autoload": {
         "psr-4": {
             "EzSystems\\EzPlatformRichTextBundle\\": "src/bundle",
-            "EzSystems\\EzPlatformRichText\\": "src/lib"
+            "EzSystems\\EzPlatformRichText\\": "src/lib",
+            "Ibexa\\FieldTypeRichText\\RichText\\": "src/lib/RichText"
```

```suggestion
            "Ibexa\\FieldTypeRichText\\RichText\\": "src/lib/RichText",
            "EzSystems\\EzPlatformRichText\\": "src/lib"
```

Adding new ones on top helps with phpstorm suggestions.